server/src/server.ts

```diff
@@ -30,6 +31,7 @@ interface extensionConfiguration {
     enable: boolean;
     maxLength: number | null;
   };
+  codeLens: boolean;
```

Comment:
Wondering whether extension config should go to its own file.

Reply:
You mean the type definition?